src/init.cpp

```diff
@@ -391,6 +391,8 @@ std::string HelpMessage(HelpMessageMode mode)
             CURRENCY_UNIT, FormatMoney(DEFAULT_TRANSACTION_MINFEE)));
     strUsage += HelpMessageOpt("-paytxfee=<amt>", strprintf(_("Fee (in %s/kB) to add to transactions you send (default: %s)"),
         CURRENCY_UNIT, FormatMoney(payTxFee.GetFeePerK())));
+    strUsage += HelpMessageOpt("-payatleastcustomfee", strprintf(_("Pay the fee set by paytxfee or settxfee considering at least 1000 bytes or the actual transaction size, whichever is higher (default: %u)"),
+        fPayAtLeastCustomFee));
```

Comment:
Nit: @jonasschnelli didn't like this. I think 

* this should hide behind ` if (showDebug)`
* this should be slowly deprecated. (i.e. GUI does not overwrite command line args at runtime and this option should be removed entirely from the GUI, and rpc after a grace period)